src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -131,14 +137,23 @@ function ReportActionItemMessageEdit(props) {
         return () => {
             // Skip if this is not the focused message so the other edit composer stays focused.
             // In small screen devices, when EmojiPicker is shown, the current edit message will lose focus, we need to check this case as well.
-            if (!isFocusedRef.current && !EmojiPickerAction.isActive(props.action.reportActionID)) {
+            // When delete modal is opened, the current edit message will lose focus, we need to check this case as well
+            if (!isActive()) {
                 return;
             }
 
+            if (EmojiPickerAction.isActive(props.action.reportActionID)) {
+                EmojiPickerAction.clearActive();
+            }
+            if (ReportActionContextMenu.isActiveReportAction(props.action.reportActionID)) {
+                ReportActionContextMenu.clearActiveReportAction();
+            }
+
             // Show the main composer when the focused message is deleted from another client
             // to prevent the main composer stays hidden until we swtich to another chat.
-            ComposerActions.setShouldShowComposeInput(true);
+            setShouldShowComposeInputKeyboardAware(true);
         };
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Please add a comment for the disabled eslint rule

Reply:
Added